udata_hydra/crawl.py

```diff
@@ -294,10 +308,18 @@ async def check_url(
                 },
             )
 
+            # Update resource status to TO_PROCESS_RESOURCE
+            await Resource.update(resource_id, data={"status": "TO_PROCESS_RESOURCE"})
+
             queue.enqueue(process_resource, check_id, is_first_check, _priority=worker_priority)
 
+            # Reset resource status to None so that it's not forbidden to be checked again
+            await Resource.update(resource_id=resource_id, data={"status": None})
```

Comment:
I think setting resource status to `None` here makes it seems like the process is one, but it continues with `process_resource`.